Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "getrandom"
 version = "0.2.15" # Also update html_root_url in lib.rs when bumping this
-edition = "2018"
-rust-version = "1.56"
+edition = "2021"
+rust-version = "1.57"
```

Comment:
Could we add a comment to the `rust-version` line (similar to what we have for `version`) that any MSRV updates should also update `README.md`, `.github/worksflows/tests.yml`, and `.clippy.toml`